packages/components/organisms/f-header/src/components/Navigation.vue

```diff
@@ -525,18 +533,25 @@ export default {
                 this.firstFocusableEl = firstFocusableEl;
                 this.lastFocusableEl = lastFocusableEl;
 
+                // Add loop-handling event listener
                 nav.addEventListener('keydown', this.keyActions);
             });
         },
 
+        toggleUserMenu () {
+            if (this.userMenuIsOpen) {
```

Comment:
Could this be `this.userMenuIsOpen ? this.closeUserMenu() : this.openUserMenu();`

Reply:
eslint isn't a massive fan of it, saying it's an "unused expression", so I've had to stick a `return` on the front which works just fine